Makefile

```diff
@@ -42,73 +57,47 @@ $(GOBIN):
 
 work: $(GOBIN)
 
-build: openstack-cloud-controller-manager cinder-provisioner cinder-flex-volume-driver cinder-csi-plugin k8s-keystone-auth client-keystone-auth octavia-ingress-controller manila-provisioner manila-csi-plugin barbican-kms-plugin magnum-auto-healer
-
-openstack-cloud-controller-manager: work $(SOURCES)
-	CGO_ENABLED=0 GOOS=$(GOOS) go build \
-		-ldflags $(LDFLAGS) \
-		-o openstack-cloud-controller-manager \
-		cmd/openstack-cloud-controller-manager/main.go
+ifeq ($(ARCH),arm32v7)
+    DEBIAN_ARCH=arm
+    GOARCH=arm
+    QEMUARCH=arm
+else ifeq ($(ARCH),arm64v8)
+    DEBIAN_ARCH=arm64
+    GOARCH=arm64
+    QEMUARCH=aarch64
+else
+    DEBIAN_ARCH=$(ARCH)
+    GOARCH=$(ARCH)
+    QEMUARCH=$(ARCH)
+endif
 
-cinder-provisioner: work $(SOURCES)
-	CGO_ENABLED=0 GOOS=$(GOOS) go build \
-		-ldflags $(LDFLAGS) \
-		-o cinder-provisioner \
-		cmd/cinder-provisioner/main.go
+build: $(addprefix build-cmd-,$(BUILD_CMDS))
 
+# Remove individual go build targets, once we migrate openlab-zuul-jobs
+# to use new build-cmd-% targets.
```

Comment:
I don't fully understand here, why the cinder-csi-plugin and manila-csi-plugin are defined separately?